instance/models/openedx_instance.py

```diff
@@ -160,10 +167,41 @@ def spawn_appserver(self):
             # Copy the current value of each setting into the AppServer, preserving it permanently:
             configuration_database_settings=self.get_database_settings(),
             configuration_storage_settings=self.get_storage_settings(),
+            configuration_extra_settings=self.get_extra_settings(),
             **instance_config
         )
         if app_server.provision():
             # If the AppServer provisioned successfully, make it the active one:
             # Note: if I call spawn_appserver() twice, and the second one provisions sooner, the first one may then
             # finish and replace the second as the active server. We are not really worried about that for now.
             self.set_appserver_active(app_server.pk)
+
+
+class OpenEdXProductionInstance(OpenEdXInstance):
+    """
+    OpenEdXProductionInstance: An OpenEdXInstance using production settings
+    """
+
+    LATEST_RELEASE = "named-release/dogwood.rc"
+
+    def save(self, **kwargs):
+        """
+        Set default field values specific to production instances before saving.
+        """
+        # Ansible-specific settings:
+        self.configuration_version = self.LATEST_RELEASE
+        self.openedx_release = self.LATEST_RELEASE
+        # Also set edx_platform_commit in case behavior of "set_field_defaults" changes
+        self.edx_platform_commit = self.LATEST_RELEASE
```

Comment:
I'd rather here that you just override `set_field_defaults` and set `openedx_release = settings.LATEST_RELEASE`, and set `configuration_version`, then just call `super().set_field_defaults()`. Setting `edx_platform_commit` is unnecessary and you can just add a test for that, rather than writing redundant code to do that "in case behavior...changes"